RELEASE_NOTES.md

```diff
@@ -1,6 +1,8 @@
-## 4.0.0-rc1
+## 4.0.0-rc2
   - Add .NET Core support for all libraries.
-  - Update to latest FSharp.Compiler.Service
+  - Update to FSCS v35.0.
```

Comment:
I've also merged `Add helpers for CustomOperationAttribute`

Reply:
Fixed